src/reputation/views.py

```diff
@@ -136,6 +139,53 @@ def get_queryset(self):
             return Withdrawal.objects.all()
         else:
             return Withdrawal.objects.filter(user=user)
+    
+    @action(
+        detail=False,
+        methods=['POST'],
+        permission_classes=[],
```

Comment:
Is this endpoint safe to be open to the public or do we want some extra permissions on it?